en/firefox/more.ftl

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the { -brand-name-mozilla } Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/more/
+
+firefox-products-are = { -brand-name-firefox } products are designed to protect your privacy
+learn-more-about = Learn more about { -brand-name-firefox } browsers and products that handle your data with respect and are built for privacy anywhere you go online.
+learn-more = Learn more
+the-history-of = The history of web browsers
+firefox-has-been = { -brand-name-firefox } has been there since almost the beginning.
+firefox-rebel-with = { -brand-name-firefox }: Rebel with a cause
+firefox-is-independent = { -brand-name-firefox } is independent and backed by the non-profit { -brand-name-mozilla }, which fights for your online rights and makes the Internet accessible to everyone, everywhere.
```

Comment:
Didn't we start using using `not-for-profit` instead of `non-profit`? Right now it is quite inconsistent on the site between migrated pages and new ones. Could you check? At least for new content, we should start the newer term, if it is the case.

Reply:
`Internet` - why is this capital initial?

Reply:
Hi Peiying! Apologies for the delay, I was out of office end of last week. Checking today.